react/week-19/lesson.md

```diff
@@ -332,57 +338,55 @@ Here we are using `Array.map` to turn an array of strings into an array of compo
 > 1. Inside the `Logo` component create a new variable called `appName` and assign it to `"Pokedex"`
 > 2. Now replace the hard-coded app name with `{appName}`. What do you see in your web browser? What would you do if you wanted to change the app name?
-> 3. Create a new component named `CaughtPokemon`. Within this component return a `<p>` with the text "Caught 0 Pokemon on" (we're going to fill in today's date in the next step)
-> 4. Create a variable named `date` within the `CaughtPokemon` component, and assign it today's date (hint: `new Date().toLocaleDateString()`). Finally, render the `date` variable after the text
+> 3. Create a new component named `CaughtPokemon`. Within this component return a `<p>` tag with the text "Caught 0 Pokemon on" (we're going to fill in today's date in the next step)
+> 4. Create a variable named `date` within the `CaughtPokemon` component, and assign it today's date (hint: `new Date().toLocaleDateString()`). Finally, render the `date` variable after the text "Caught 0 Pokemon on"
 > 5. Render the `CaughtPokemon` component within the `App` component (below `BestPokemon`)
 > 6. Within the `BestPokemon` component, create a variable named `pokemonNames` and assign it to an array with some Pokemon names (e.g. `['Squirtle', 'Bulbasaur', 'Charmander']`)
 > 7. Change the `BestPokemon` component to return a `<ul>` element instead of a `<p>` element
 > 8. Now use the `.map()` method on the `pokemonNames` variable to loop over each name and return a `<li>` element for each (hint: look at the mentors list example above)
 
 ## Keys
 
-You may have noticed that we are now seeing a red error message in the Dev Tools: `Warning: Each child in a list should have a unique "key" prop.`. This error happens when you use `Array.map` to return a list of elements ([interactive example](https://codesandbox.io/s/pwp8ox4kn0)):
+You may have noticed that we are now seeing a red error message in the Dev Tools: `Warning: Each child in a list should have a unique "key" prop.`. This error happens when you use `Array.map` to return a list of elements ([interactive example](https://codesandbox.io/s/846y8rlrnl)):
 
 ```js
 const mentors = ['Ali', 'Kash', 'Davide', 'German', 'Gerald']
 
 const List = () => (
   <ul>
     {mentors.map((name, index) => {
-      return (
-        <li key={index}>{name}</li>
-      )
+      return <li key={index}>{name}</li>;
     })}
   </ul>
-)
+);
 ```
 
-Here we have added a `key` prop the to element. A documentation page explaining in more depth is in the further reading section but basically the `key` prop has a special meaning in React because it is used internally to keep track of which element in the list is which.
+Here we have added a `key` property to the `li` element. A documentation page explaining in more depth is in the further reading section but basically the `key` prop has a special meaning in React because it is used internally to keep track of which element in the list is which.
```

Comment:
My feeling generally we refer to `key` as a "prop", so I think for consistency I'd prefer to keep that.